Hazel/vendor/Glad/include/KHR/khrplatform.h

```diff
@@ -90,12 +90,20 @@
  *                                  int arg2) KHRONOS_APIATTRIBUTES;
  */
 
+#if defined(__SCITECH_SNAP__) && !defined(KHRONOS_STATIC)
```

Comment:
This more like a question, but why are modifying automatically generated code? What are you trying to accomplish?

Reply:
These are changes that are done by @marian-bielcik , these are indeed auto generated files. Since the commit (https://github.com/TheCherno/Hazel/pull/82/commits/0853563d568f55f7449b63b50f4ab48c23c708c2) states an update of the OpenGL code, I suspect marian did an update of the glad module and not changed these lines manualy...

Reply:
Which files are automatically generated? From what project are they generated from?

Reply:
They are from Hazel/vendor/Glad, generated by https://glad.dav1d.de/